src/ElasticsearchDAO.py

```diff
@@ -7,17 +7,17 @@ class ElasticsearchDAO(object):
     def __init__(self):
         super().__init__()
 
-    def writeToElasticsearch(self, datasets):
+    def write_To_Elasticsearch(self, datasets):
         ELASTICSEARCH_API_BASE_URL = os.environ.get('ELASTICSEARCH_API_BASE_URL')\
                                      if os.environ.get('ELASTICSEARCH_API_BASE_URL') is not None else 'http://localhost'
         esresult = requests.get(ELASTICSEARCH_API_BASE_URL + '/dataassets/_search?size=10000')
-        existingDocs = json.loads(esresult.text)['hits']['hits']
+        existing_Docs = json.loads(esresult.text)['hits']['hits']
         document = ''
         for dataset in datasets:
             found = False
-            lineObj = {}
-            lineIndexObj = {}
-            for x in existingDocs:
+            line_Obj = {}
+            line_IndexObj = {}
+            for x in existing_Docs:
```

Comment:
When you rename a variable, please make sure that the variable name is updated everywhere that it is called to make sure the rename does not result in any bugs.

For example, here lineObj is renamed to line_Obj in line 18, but various places in the script (e.g. in line 23 and 28) still use lineObj.